vmdb/config/routes.rb

```diff
@@ -1887,6 +1887,9 @@
   match '/api/*suffix' => 'api#update',  :format => 'json', :via => [:post, :put, :patch]
   match '/api/*suffix' => 'api#destroy', :format => 'json', :via => [:delete]
 
+  # OPTIONS requests for pre-flight checks
+  match '*path', :controller => 'api', :action => 'handle_options_request', :constraints => {:method => 'OPTIONS'}
```

Comment:
Will limit that to API.

But the point is we might want to limit it to certain resources under API or even further.